lib/bolt/plan_creator.rb

```diff
@@ -119,6 +126,33 @@ def self.yaml_plan(plan_name)
       YAML
     end
 
+    def self.yaml_script_plan(_plan_name, script)
```

Comment:
```suggestion
    def self.yaml_script_plan(script)
```

Or add it to the plan output somewhere.